pkgs/by-name/re/responsively-desktop/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  appimageTools,
+}:
+
+appimageTools.wrapType2 rec {
+  pname = "responsively-desktop";
+  version = "1.15.0";
+
+  src = fetchurl {
+    url = "https://github.com/responsively-org/responsively-app-releases/releases/download/v${version}/ResponsivelyApp-${version}.AppImage";
+    hash = "sha256-BkljY8Il45A2JbsLgQbjsxCy0lnFZvtpc5HzvI1nwWk=";
+  };
+
+  appimageContents = appimageTools.extract {
+    inherit pname version src;
+  };
+
+  extraInstallCommands = ''
+    install -m 444 -D ${appimageContents}/responsivelyapp.desktop $out/share/applications/responsivelyapp.desktop
+    install -m 444 -D ${appimageContents}/usr/share/icons/hicolor/512x512/apps/responsivelyapp.png \
+      $out/share/icons/hicolor/512x512/apps/responsivelyapp.png
+    substituteInPlace $out/share/applications/responsivelyapp.desktop \
+      --replace 'Exec=AppRun' 'Exec=${pname}'
+  '';
+
+  meta = {
+    description = "A modified web browser that helps in responsive web development";
```

Comment:
```suggestion
    description = "Modified web browser that helps in responsive web development";
```

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/307444/commits/f4e58eef423896cbc497b7d5abdc14ae6284c7ee